instrumentation/opentelemetry-instrumentation-wsgi/setup.py

```diff
@@ -11,6 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+
+# DO NOT EDIT. THIS FILE WAS AUTOGENERATED FROM templaes/wsgi.
```

Comment:
looks like a typo sneaked into the generated code, though it doesnt appear in the generator 🤔

Reply:
Yea, looks like I updated the generator but didn't run it before checking in.